service/src/main/java/gov/nasa/pds/api/registry/search/RegistrySearchRequestBuilder.java

```diff
@@ -106,6 +106,39 @@ public BoolQuery.Builder getQueryBuilder() {
     return this.queryBuilder;
   }
 
+  /**
+   * Applies a common set of constraints and other build options which generally apply to any endpoint which queries
+   * OpenSearch for a result-set of multiple products.
+   * @param includeFieldNames - which properties to include in the results (JSON format, not OpenSearch format)
+   * @param queryString - a querystring (q=) to constrain the result-set by
+   * @param keywords - a set of keyword matches to <exact behaviour TBD - not implemented yet>
+   * @param pageSize - the page size to use for pagination
+   * @param sortFieldNames - the fields by which results are sorted (ascending), from highest to lowest priority
+   * @param searchAfterFieldValues - the values corresponding to the sort fields, for pagination
+   * @param noSupersededProducts - whether to exclude superseded products from the result set
+   */
+  public RegistrySearchRequestBuilder applyMultipleProductsDefaults(
+          List<String> includeFieldNames,
+          String queryString,
+          List<String> keywords,
+          Integer pageSize,
+          List<String> sortFieldNames,
+          List<String> searchAfterFieldValues,
+          Boolean noSupersededProducts
+  ) throws UnparsableQParamException, SortSearchAfterMismatchException {
+    this
+      .fieldsFromStrings(includeFieldNames)
+      .constrainByQueryString(queryString)
+      .addKeywordsParam(keywords)
+      .paginate(pageSize, sortFieldNames, searchAfterFieldValues);
+
+    if (noSupersededProducts) {
```

Comment:
I would not have had this in the default since it is not default anymore as sometimes we have it and sometimes we don't but this is very minor.

Reply:
the no-superseded filter?

I thought it was considered default, but overridable by a yet-to-be-implemented qparam?